generator/src/main/java/com/scottlogic/deg/generator/CommandLine/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Contains classes that hold the options passed in on the command line.
+ */
+package com.scottlogic.deg.generator.CommandLine;
```

Comment:
What is this file?

Reply:
I'm guessing this is simply to add a javadoc to the package? I'd prefer we didn't go to this level of documentation, and leave the code as self-documenting as possible.

Reply:
okay, removed.